src/main/java/organice/logic/parser/ListCommandParser.java

```diff
@@ -0,0 +1,65 @@
+package organice.logic.parser;
+
+import static organice.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static organice.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import java.util.NoSuchElementException;
+
+import java.util.stream.Stream;
+
+import organice.logic.commands.ListCommand;
+import organice.logic.parser.exceptions.ParseException;
+import organice.model.person.Type;
+
+/**
+ * Parses input arguments and creates a new ListCommand object
+ */
+public class ListCommandParser implements Parser<ListCommand> {
+
+    /**
+     * Returns the {@code Type} of person in the given {@code ArgumentMultimap}
+     * @throws ParseException if the type is not specified correctly in the input arguments
+     */
+    private static Type parseType(ArgumentMultimap argumentMultimap) throws ParseException {
+        try {
+            return ParserUtil.parseType(argumentMultimap.getValue(PREFIX_TYPE).get());
+        } catch (NoSuchElementException ex) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ListCommand.MESSAGE_USAGE));
+        }
+    }
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ListCommand
+     * and returns a ListCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ListCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TYPE);
+        arePrefixesPresent(argMultimap);
```

Comment:
no need to check whether its present, cos u have deal with the `ParseException`

Reply:
Removed `arePrefixesPresent(argMultimap);`.